### PR TITLE
Update FirebasePluginMessageReceiver.h

### PR DESCRIPTION
**Pull request by amilamen**
I've an issue when I try to build my project on XCode 11.

I've fixed using XCode suggestion and it works.

## PR Type
What kind of change does this PR introduce?

- [ x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation  changes
- [ ] Other... Please describe:

## PR Checklist
For bug fixes / features, please check if your PR fulfills the following requirements:

- [ ] Testing has been carried out for the changes have been added
- [ ] Regression testing has been carried out for existing functionality
- [ ] Docs have been added / updated

## Does this PR introduce a breaking change?
- [ ] Yes
- [x ] No

## Other information

### CODE REVIEW

File: src/ios/FirebasePluginMessageReceiver.h [hunk 1]
```diff
@@ -1,5 +1,6 @@
-#import <Foundation/Foundation.h>
+#import "FirebasePluginMessageReceiver.h"
```
Comment (dpa99c) [inline src/ios/FirebasePluginMessageReceiver.h:1]: This does not make sense - why would this file import itself?

Comment (amilamen) [reply src/ios/FirebasePluginMessageReceiver.h:1]: Thanks sir for your reply. I agree with you but I don't know why XCode suggest me this alternative and when I do it, it solves my bug automatically.

**Comment by dpa99c** [comment]
Sorry but I can't accept this pull request.
The [example project](https://github.com/dpa99c/cordova-plugin-firebasex-test) contain the plugin builds fine in Xcode 11.6 and Xcode 12.0 Beta 6 so I'm satisfied that the changes in this PR are not necessary.

**Comment by amilamen** [comment]
Capture d’écran 2020-09-08 à 09 28 03
Capture d’écran 2020-09-08 à 09 28 28
Capture d’écran 2020-09-08 à 09 29 48
Capture d’écran 2020-09-08 à 09 30 06

Screenshots of all infos you need.

**Comment by amilamen** [comment]
> Sorry but I can't accept this pull request.
> The [example project](https://github.com/dpa99c/cordova-plugin-firebasex-test) contain the plugin builds fine in Xcode 11.6 and Xcode 12.0 Beta 6 so I'm satisfied that the changes in this PR are not necessary.

I hope your response and thanks for all sir.

**Comment by amilamen** [comment]
Capture d’écran 2020-09-08 à 09 36 48

 You can show filename in top of screenshot.

**Pull request closed without merging**